# Techno-Economic Analysis: Hydrogen and Acetylene Black Production from Renewable Natural Gas via Microwave Plasma Reactors

## Executive Summary

This analysis evaluates the techno-economic viability of producing hydrogen gas and acetylene black from renewable natural gas (RNG) using Pentacarbon's proprietary microwave plasma reactor technology.

### Key Results

**Financial Performance:**
- **NPV (20 years, 15% discount):** $XX million
- **IRR:** XX%
- **Payback Period:** XX years
- **ROI (Year 3):** XX%
- **LCOH:** $XX/kg H2

**Production Capacity**
- **Hydrogen:** 393.1 kg/hr (3,100 tonnes/year)
- **Acetylene Black:** 824.6 kg/hr (6,500 tonnes/year)
- **Capacity Factor:** 91.7%

**Environmental Performance**
- **CO2 Intensity:** 0.12 kg CO2eq/kg H2 (99.9% reduction vs. SMR)
- **Water Intensity:** 0.6kg H2O/kg H2 (86.7% reduction vs. SMR)
- **Energy Source:** Renewable electricity
- **Feedstock Source:** Landfill methane that would otherwise be released to atmosphere

**Economic Advantages:**
- Strong profitability with conservative pricing
- Premium co-product (acetylene black) provides 40% of revenue with downstream sustainability applications
- Near-zero emissions using renewable energy
- Minimal water consumption
- Utilizes waste/renewable methane feedstock

---

## Background

### The Hydrogen Challenge

Hydrogen is critical for:
- **Industrial decarbonization** (ammonia, steel, refining)
- **Clean transportation** (fuel cells)
- **Energy storage** (power-to-gas)
- **Chemical Manufacturing**

However, **96% of hydrogen is produced from fossil fuels via steam methane reforming (SMR)**, emitting ~845 million tonnes CO2 annually worldwide (9-10kg CO2 per kg H2).

### The Carbon Black Opportunity

Acetylene black is a **highly conductive carbon black** used in:
- Lithium-ion batteries
- Supercapacitors
- Advanced electronics
- Specialty coatings

It commands **2-3x the price** of commodity carbon blacks used in tires.

### The Solution: Microwave Plasma Conversion

**Core Reaction:** CH4 &rarr; C(s) + 2H2

Using **microwave-induced plasma at 2000C** we crack methane into:
- **Hydrogen gas** (99.9% purity after separation)
- **Acetylene black** (premium conductive solid carbon)

This process:
- Eliminates CO2 emissions  (when using renewable electricity)
- Minimizes water consumption (87% less than industry standard SMR)
- Produces valuable co-product with downstream tech applications
- Uses renewable natural gas feedstock that would otherwise contribute to global emissions

---

## Process Description

### Simplified Process Flow

<pre>
                                                                                            ← Methane Recycle
                                                                                                    ↑
RNG Feed → Pentacarbon Microwave Reactor → Quench and Compression → Acetylene Absorption → Hydrogen Separation →
                                                    ↓                       ↓
                                             ← Water Recycle          Conversion to
                                                                      Acetylene Black → Solids Processing →
</pre>

### Process Steps

**1. Feed**
- **Flow rate:** 1923 kg/hr renewable natural gas
- **Composition:** 90.5% CH4, 7.8% CO2, 1.3% N2, trace H2S
- **Treatment:** Heat to 38C, compress to 2 bar

**2. Microwave Reactor (R1):**
- **Temperature:** 2000C (plasma conditions)
- **Pressure:** 2.07 bar
- **Power Requirement:** 13,400.0 kW
- **Conversion:** 90%
- **Products:** H2, acetylene (C2H2), ethylene (C2H4), unreacted CH4

**Quench and Compression**
- Rapid water quench to 114C
- Compression to 18.8 bar for absorption

**4. Acetylene Absorption**
- Acetylene absorbed in water solvent
- Separates acetylene-rich stream from H2-rich stream

**5. Conversion to Acetylene Black**
- **Acetylene-rich stream
- **Temperature:** 2000C
- **Reaction:** C2H2 &rarr; 2C(s) + H2
- **Product:** Acetylene Black (premium carbon black)
- Additional hydrogen recovered and recycled

**6. Hydrogen Purification**
- Pressure Swing Absoroption (PSA)
- **Final Purity:** 99.9% H2
- **Production:** 393.1 kg/hr

**7. Solids Processing**
- Solid carbon collected via cyclone
- **Grade:** Acetylene Black (premium conductive)
- **Production:** 824.6 kg/hr

---

## Key Process Specifications

### Feed Composition (ASPEN Stream: Feed (1))


| Component | kg/hr | % by mass |
|-----------|-------|-----------|
| Methane (CH₄) | 1741.4 | 90.5% |
| Carbon Dioxide (CO₂) | 149.3 | 7.8% |
| Nitrogen (N₂) | 25.3 | 1.3% |
| H₂S (trace) | 0.02 | <0.01% |
| **Total** | **1923** | **100%** |

### Product Streams (ASPEN Stream: HYDPROD(34), CARBON(28))

| Product | kg/hr | tonnes/day | tonnes/year |
|---------|-------|------------|-------------|
| Hydrogen (99.9%) | 393.1 | 9.43 | 3,100 |
| Acetylene Black | 824.6 | 19.79 | 6,500 |

### Energy Consumption

| Utility | Consumption | Annual Cost |
|---------|-------------|-------------|
| Electricity (total) | 14,480 kW | $8,365,843 |
| Cooling Water | 237,847 L/hr | $51,861 |
| Process Water | 252 L/hr | $539 |

---

## Based on an Original Publication at the University of Pennsylvania

**Title:** "Natural Gas Conversion into Clean Hydrogen and Carbon for Li-Ion Battery Electrodes Using Renewable Energy"

**Authors:** Adam Ritter, Sydney Levine, Skylar Cannon

**Advisors:** Dr. Matthew Targett, Dr. Warren Seider

**Institution:** University of Pennsylvania, Department of Chemical and Biomolecular Engineering

**Year:** 2022

**Note:** This analysis uses ASPEN Plus simulation results from our original design project. I've taken this opportunity to expand and refine the economic analysis, siting assessment, and uncertainty analysis.

[Link to thesis](https://repository.upenn.edu/entities/publication/89c4a34b-5df2-4229-b0c5-94324f8dcfd4)


In [1596]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import plotly.graph_objects as go
import plotly.express as px
from plotly.subplots import make_subplots
import numpy_financial as npf
import warnings
warnings.filterwarnings("ignore")

sns.set_style("whitegrid")
plt.rcParams['figure.figsize'] = (12, 6)
plt.rcParams['font.size'] = 11

print("✓ Libraries imported successfully")
print(f"✓ Analysis Date: November 2025")
print(f"✓ Based on: Penn CBE Senior Design Project 2022")
print(f"✓ Process parameters are sourced from the original project data. Economic assumptions have been updated to reflect 2025 market conditions.")

✓ Libraries imported successfully
✓ Analysis Date: November 2025
✓ Based on: Penn CBE Senior Design Project 2022
✓ Process parameters are sourced from the original project data. Economic assumptions have been updated to reflect 2025 market conditions.


In [1597]:
# load csv data, define a function to extract values, display dataframes
process_params = pd.read_csv('data/process_parameters.csv')
econ_assumptions = pd.read_csv('data/economic_assumptions.csv')
sensitivity_ranges = pd.read_csv('data/sensitivity_ranges.csv')

def get_value(df, param_name):
    return df[df['parameter'] == param_name]['value'].values[0]

print("="*80)
print("PROCESS PARAMETERS (from ASPEN Plus Simulation)")
print("="*80)
display(process_params)

print("\n" + "="*80)
print("ECONOMIC ASSUMPTIONS (from Financial Analysis)")
print("="*80)
display(econ_assumptions)

print("\n" + "="*80)
print("SENSITIVITY ANALYSIS RANGES")
print("="*80)
display(sensitivity_ranges)

PROCESS PARAMETERS (from ASPEN Plus Simulation)


parameter      value     unit  \
0                     rng_feed_rate    1923.00    kg/hr   
1          hydrogen_production_rate     393.10    kg/hr   
2   acetylene_black_production_rate     824.60    kg/hr   
3                process_water_rate     252.00     L/hr   
4                cooling_water_rate  237847.00     L/hr   
5               co2_production_rate       4.50    kg/hr   
6                    plant_lifetime      20.00    years   
7                   capacity_factor      91.00  percent   
8               reactor_temperature    2000.00  celsius   
9                  reactor_pressure       2.07      bar   
10                power_requirement   14481.00       kW   

                       source                             notes  
0           ASPEN FEED stream  renewable natural gas input rate  
1        ASPEN HYDPROD stream         Purified hydrogen product  
2         ASPEN CARBON stream              Solid carbon product  
3            Absorption tower               Process water input  
4      Cooler 1-3 and Flash 1               Cooling water input  
5    ASPEN net CO2 production        CO2 byproduct from reactor  
6          Financial analysis         Economic evaluation basis  
7           industry standard  percentage actual operating time  
8          ASPEN R1OUT stream               Pentacarbon reactor  
9          ASPEN R1OUT stream        Slightly above atmospheric  
10         process simulation     Reactor + compression + pumps


ECONOMIC ASSUMPTIONS (from Financial Analysis)


parameter         value             unit  \
0                       rng_price  1.800000e-01           USD/kg   
1               electricity_price  7.000000e-02          USD/kWh   
2                  hydrogen_price  4.000000e+00           USD/kg   
3           acetylene_black_price  2.200000e+00           USD/kg   
4             cooling_water_price  2.700000e-05            USD/L   
5             process_water_price  2.700000e-04            USD/L   
6                   discount_rate  1.500000e+01          percent   
7               depreciation_rate  8.000000e+00          percent   
8           average_salvage_value  1.000000e+01          percent   
9   general_variable_expense_rate  1.155000e+01          percent   
10            operators_per_shift  1.000000e+00  operators/shift   
11                 shifts_per_day  5.000000e+00       shifts/day   
12            direct_hourly_wages  4.000000e+01         USD/hour   
13        equipment_purchase_cost  1.400000e+07              USD   
14             bare_module_factor  3.210000e+00              NaN   
15           average_salvage_rate  1.000000e+01          percent   

                                        notes  \
0                  Renewable natural gas 2025   
1                        Industrial rate 2025   
2                        Wholesale price 2025   
3                   Premium carbon black 2025   
4                                Utility cost   
5                                Utility cost   
6                             WACC assumption   
7                  straight-line depreciation   
8             average end of life asset value   
9   selling/transfer/admin/research/incentive   
10                            Plant Operators   
11                             Shifts per Day   
12                               Direct Wages   
13              Total equipment purchase cost   
14                    Bare module cost factor   
15            average end of life asset value   

                                    calculation_basis  
0                                  Updated from $0.16  
1                                  Updated from $0.07  
2                                  Updated from $3.18  
3                                  Updated from $2.00  
4                               Industrial water rate  
5                                  Treated water rate  
6                                 For NPV calculation  
7                         Annual depreciation expense  
8                     Percent of initial capital cost  
9                            percent of total revenue  
10                      number of operators per shift  
11                           number of shifts per day  
12                           hourly wage per operator  
13                       Sum of major equipment items  
14  Includes installation/piping/instrumentation/e...  
15                    Percent of initial capital cost


SENSITIVITY ANALYSIS RANGES


parameter   base_value    low_value   high_value     unit  \
0      electricity_price         0.07         0.04         0.10  USD/kWh   
1         hydrogen_price         4.00         2.00         6.00   USD/kg   
2  acetylene_black_price         2.00         1.00         3.00   USD/kg   
3              rng_price         0.16         0.10         0.22   USD/kg   
4            capex_total  45000000.00  30000000.00  60000000.00      USD   
5      carbon_conversion        90.00        85.00        95.00  percent   
6        capacity_factor        91.70        85.00        95.00  percent   
7          discount_rate        15.00        10.00        20.00  percent   

                         notes  
0      Regional variation ±40%  
1  Market volatility scenarios  
2   Grade and market dependent  
3     Feedstock cost variation  
4             ±33% contingency  
5    Process performance range  
6      Operational reliability  
7   Financial assumption range

In [1598]:
# extract parameters for key metric calculations

# process parameters
# hydrogen_production_rate = get_value(process_params, 'hydrogen_production_rate')  # kg/hr
# acetylene_black_production_rate = get_value(process_params, 'acetylene_black_production_rate')  # kg/hr
# annual_operating_days = get_value(process_params, 'annual_operating_days')
# plant_lifetime = get_value(process_params, 'plant_lifetime')
# capacity_factor = get_value(process_params, 'plant_capacity_factor') / 100
# energy_per_kg_h2 = get_value(process_params, 'energy_per_kg_h2')
# co2_per_kg_h2 = get_value(process_params, 'co2_per_kg_h2')
# water_per_kg_h2 = get_value(process_params, 'water_per_kg_h2')

# prices
# h2_price = get_value(econ_assumptions, 'hydrogen_price')
# cb_price = get_value(econ_assumptions, 'acetylene_black_price')
# electricity_price = get_value(econ_assumptions, 'electricity_price')
# rng_price = get_value(econ_assumptions, 'rng_price')
# discount_rate = get_value(econ_assumptions, 'discount_rate') / 100

# calculate economic parameters
# capex = get_value(econ_assumptions, 'capex_total')
# h2_revenue = get_value(econ_assumptions, 'annual_hydrogen_revenue')
# cb_revenue = get_value(econ_assumptions, 'annual_acetylene_black_revenue')
# #total_revenue = get_value(econ_assumptions, 'total_annual_revenue')
# variable_costs = get_value(econ_assumptions, 'total_annual_variable_costs')
# gross_margin = get_value(econ_assumptions, 'annual_gross_margin')
# npv = get_value(econ_assumptions, 'npv_20_year')
# irr = get_value(econ_assumptions, 'irr')
# payback = get_value(econ_assumptions, 'payback_period')
# lcoh = get_value(econ_assumptions, 'lcoh')


In [1599]:
# #calculate key metrics and print summary

# #process
# operating_hours = annual_operating_days * 24
# annual_h2_rate = h2_rate * operating_hours

# h2_revenue = h2_price * h2_annual
# cb_revenue = cb_price * cb_annual

# print("="*80)
# print("KEY PERFORMANCE METRICS")
# print("="*80)

# print(f"\n📊 PRODUCTION CAPACITY:")
# print(f"   Hydrogen: {h2_rate:,.1f} kg/hr = {h2_annual/1000:,.1f} tonnes/year")
# print(f"   Acetylene Black: {cb_rate:,.1f} kg/hr = {cb_annual/1000:,.1f} tonnes/year")
# print(f"   Capacity Factor: {capacity_factor*100:.1f}%")
# print(f"   Operating Hours: {operating_hours:,.0f} hrs/year")

# print(f"\n💰 FINANCIAL PERFORMANCE:")
# print(f"   Total CapEx: ${capex/1e6:.2f} million")
# print(f"   Annual Revenue: ${total_revenue/1e6:.2f} million")
# print(f"   Annual Variable Costs: ${variable_costs/1e6:.2f} million")
# print(f"   Gross Margin: ${gross_margin/1e6:.2f} million ({gross_margin/total_revenue*100:.1f}%)")
# print(f"   NPV (20 yr, 15%): ${npv/1e6:.2f} million")
# print(f"   IRR: {irr:.2f}%")
# print(f"   Payback: {payback:.1f} years")
# print(f"   LCOH: ${lcoh:.2f}/kg H₂")

# print(f"\n💵 MARKET PRICES:")
# print(f"   Hydrogen: ${h2_price}/kg (wholesale)")
# print(f"   Acetylene Black: ${cb_price}/kg (premium grade)")
# print(f"   Electricity: ${electricity_price}/kWh")
# print(f"   RNG Feedstock: ${rng_price}/kg")

# print(f"\n🌍 ENVIRONMENTAL:")
# print(f"   CO₂ Intensity: {co2_per_kg_h2} kg CO₂eq/kg H₂")
# print(f"   Water Intensity: {water_per_kg_h2} kg H₂O/kg H₂")
# print(f"   Energy Intensity: {energy_per_kg_h2} kWh/kg H₂")

# # revenue breakdown
# h2_revenue_pct = h2_revenue / total_revenue * 100
# cb_revenue_pct = cb_revenue / total_revenue * 100
# print(f"\n📈 REVENUE MIX:")
# print(f"   Hydrogen: {h2_revenue_pct:.1f}% (${h2_revenue/1e6:.2f}M)")
# print(f"   Acetylene Black: {cb_revenue_pct:.1f}% (${cb_revenue/1e6:.2f}M)")



In [1600]:
# Extract process parameters and economic inputs

#process
rng_feed_rate = get_value(process_params, 'rng_feed_rate')  # kg/hr
h2_rate = get_value(process_params, 'hydrogen_production_rate')  # kg/hr
cb_rate = get_value(process_params, 'acetylene_black_production_rate')  # kg/hr
co2_rate = get_value(process_params, 'co2_production_rate')  # kg/hr
process_water_rate = get_value(process_params, 'process_water_rate')  # L/hr
cooling_water_rate = get_value(process_params, 'cooling_water_rate')  # L/hr
energy_rate = get_value(process_params, 'power_requirement')  # kW
capacity_factor = get_value(process_params, 'capacity_factor') # percent
plant_lifetime = get_value(process_params, 'plant_lifetime') # years

#econ
h2_price = get_value(econ_assumptions, 'hydrogen_price') # usd per kg
cb_price = get_value(econ_assumptions, 'acetylene_black_price') # usd per kg
electricity_price = get_value(econ_assumptions, 'electricity_price') # usd per kWh
rng_price = get_value(econ_assumptions, 'rng_price') # usd per kg
cooling_water_price = get_value(econ_assumptions, 'cooling_water_price') # usd per L
process_water_price = get_value(econ_assumptions, 'process_water_price') # usd per L
equipment_purchase_cost = get_value(econ_assumptions, 'equipment_purchase_cost') # usd
bare_module_factor = get_value(econ_assumptions, 'bare_module_factor') # to calculate total installed cost
discount_rate = get_value(econ_assumptions, 'discount_rate') / 100 # as decimal
general_variable_expense_rate = get_value(econ_assumptions, 'general_variable_expense_rate') / 100 # as decimal
salvage_rate = get_value(econ_assumptions, 'average_salvage_rate') # percent
operators_per_shift = get_value(econ_assumptions, 'operators_per_shift') # number of operators per shift
shifts_per_day = get_value(econ_assumptions, 'shifts_per_day') # number of shifts per day
direct_hourly_wages = get_value(econ_assumptions, 'direct_hourly_wages') # usd per operator per hour

In [1601]:
# calculate initial metrics and display

#process
annual_operating_days = 365 * capacity_factor / 100
annual_operating_hours = annual_operating_days * 24
hydrogen_production = h2_rate * annual_operating_hours  # kg/year
acetylene_black_production = cb_rate * annual_operating_hours  # kg/year
co2_per_h2 = co2_rate / h2_rate  # kg CO2/kg H2
h2o_per_h2 = process_water_rate / h2_rate  # kg H2O/kg H2
energy_per_h2 = energy_rate / h2_rate  # kWh/kg H2

#econ
#revenue
h2_revenue = h2_price * hydrogen_production # usd/year
cb_revenue = cb_price * acetylene_black_production # usd/year
total_revenue = h2_revenue + cb_revenue # usd/year

#variable costs
electricity_cost = electricity_price * (energy_rate * annual_operating_hours) # usd/year
rng_cost = rng_price * rng_feed_rate * annual_operating_hours # usd/year
cooling_water_cost = cooling_water_price * cooling_water_rate * annual_operating_hours # usd/year
process_water_cost = process_water_price * process_water_rate * annual_operating_hours # usd/year
general_variable_expenses = general_variable_expense_rate / 100 * total_revenue # usd/year, includes selling/transfer expenses, research, admin expenses, management incentives
variable_opex = electricity_cost + rng_cost + process_water_cost + general_variable_expenses # usd/year

#fixed costs
capex = equipment_purchase_cost * bare_module_factor # usd, total installed cost
labor_cost = operators_per_shift * shifts_per_day * direct_hourly_wages * annual_operating_days # usd/year
operations_cost = .21 * direct_hourly_wages * operators_per_shift * shifts_per_day * annual_operating_days + 125000 # 21% of direct wages + 125,000 usd/year technical assistance
maintenance_cost = .1035 * capex # 10.35% of capex, including salaries and benefits, materials and services, overhead
operating_overhead = .0103 * capex # 1.03% of capex, including general plant overhead, mechanical department services, employee relations, business services
taxes_insurance = .02 * capex # 2% of capex for taxes and insurance
fixed_opex = labor_cost + operations_cost + maintenance_cost + operating_overhead + taxes_insurance # usd/year

crf = discount_rate * (1 + discount_rate) ** plant_lifetime / ((1 + discount_rate) ** plant_lifetime - 1)  # capital recovery factor
annual_capex = capex * crf # usd/year
gross_margin = total_revenue - variable_opex # usd/year
lcoh = (variable_opex + fixed_opex + annual_capex - cb_revenue) / hydrogen_production  # usd/kg H2

print("="*80)
print("INITIAL PERFORMANCE METRICS")
print("="*80)

print(f"\n📊 PRODUCTION CAPACITY:")
print(f"   Hydrogen: {h2_rate:,.1f} kg/hr = {hydrogen_production/1000:,.1f} tonnes/year")
print(f"   Acetylene Black: {cb_rate:,.1f} kg/hr = {acetylene_black_production/1000:,.1f} tonnes/year")
print(f"   Capacity Factor: {capacity_factor:.1f}%")
print(f"   Operating Hours: {annual_operating_hours:,.0f} hrs/year")

print(f"\n💰 FINANCIAL PERFORMANCE:")
print(f"   Total CapEx: ${capex/1e6:.2f} million")
print(f"   Annual Revenue: ${total_revenue/1e6:.2f} million")
print(f"   Annual Variable Costs: ${variable_opex/1e6:.2f} million")
print(f"   Gross Margin: ${gross_margin/1e6:.2f} million ({gross_margin/total_revenue*100:.1f}%)")
print(f"   LCOH: ${lcoh:.2f}/kg H₂")

print(f"\n💵 MARKET PRICES:")
print(f"   Hydrogen: ${h2_price}/kg (wholesale)")
print(f"   Acetylene Black: ${cb_price}/kg (premium grade)")
print(f"   Electricity: ${electricity_price}/kWh")
print(f"   RNG Feedstock: ${rng_price}/kg")

print(f"\n🌍 ENVIRONMENTAL:")
print(f"   CO₂ Intensity: {co2_per_h2:.2f} kg CO₂eq/kg H₂")
print(f"   Water Intensity: {h2o_per_h2:.2f} kg H₂O/kg H₂")
print(f"   Energy Intensity: {energy_per_h2:.2f} kWh/kg H₂")

INITIAL PERFORMANCE METRICS

📊 PRODUCTION CAPACITY:
   Hydrogen: 393.1 kg/hr = 3,133.6 tonnes/year
   Acetylene Black: 824.6 kg/hr = 6,573.4 tonnes/year
   Capacity Factor: 91.0%
   Operating Hours: 7,972 hrs/year

💰 FINANCIAL PERFORMANCE:
   Total CapEx: $44.94 million
   Annual Revenue: $27.00 million
   Annual Variable Costs: $10.87 million
   Gross Margin: $16.12 million (59.7%)
   LCOH: $3.13/kg H₂

💵 MARKET PRICES:
   Hydrogen: $4.0/kg (wholesale)
   Acetylene Black: $2.2/kg (premium grade)
   Electricity: $0.07/kWh
   RNG Feedstock: $0.18/kg

🌍 ENVIRONMENTAL:
   CO₂ Intensity: 0.01 kg CO₂eq/kg H₂
   Water Intensity: 0.64 kg H₂O/kg H₂
   Energy Intensity: 36.84 kWh/kg H₂


In [1602]:
# Revenue breakdown
revenue_data = pd.DataFrame({
    'Product': ['Hydrogen', 'Acetylene Black'],
    'Annual Revenue': [h2_revenue, cb_revenue],
    'Revenue %': [h2_revenue / total_revenue * 100, cb_revenue / total_revenue * 100]
})

fig = make_subplots(
    rows=1, cols=2,
    subplot_titles=('Annual Revenue by Product', 'Revenue Mix'),
    specs=[[{'type': 'bar'}, {'type': 'pie'}]]
)

# Bar chart
fig.add_trace(
    go.Bar(
        x=revenue_data['Product'],
        y=revenue_data['Annual Revenue'],
        text=[f'${x/1e6:.1f}M' for x in revenue_data['Annual Revenue']],
        textposition='outside',
        marker_color=['#3498db', '#95a5a6'],
        showlegend=False
    ),
    row=1, col=1
)

# Pie chart
fig.add_trace(
    go.Pie(
        labels=revenue_data['Product'],
        values=revenue_data['Annual Revenue'],
        marker_colors=['#3498db', '#95a5a6'],
        textinfo='label+percent',
        showlegend=False
    ),
    row=1, col=2
)

fig.update_xaxes(title_text="Product", row=1, col=1)
fig.update_yaxes(title_text="Annual Revenue (USD)", row=1, col=1)
fig.update_layout(height=400, showlegend=False, title_text="Revenue Analysis")

#add 10% space to ensure text fits
max_value = revenue_data['Annual Revenue'].max()
fig.update_yaxes(range=[0, max_value * 1.1])
fig.show()

print(f"\n💡 Key Insight:")
print(f"   Acetylene black provides {(cb_revenue / total_revenue) * 100:.1f}% of revenue despite being")
print(f"   a 'co-product'. This highly-conductive solid carbon by-product is critical to project economics.")


💡 Key Insight:
   Acetylene black provides 53.6% of revenue despite being
   a 'co-product'. This highly-conductive solid carbon by-product is critical to project economics.


In [1603]:

# Create cost breakdown
cost_data = pd.DataFrame({
    'Cost Category': ['RNG Feedstock', 'Electricity', 'Cooling Water', 'Process Water'],
    'Annual Cost': [rng_cost, electricity_cost, cooling_water_cost, process_water_cost],
    'Percent of Total': [
        rng_cost/variable_opex*100,
        electricity_cost/variable_opex*100,
        cooling_water_cost/variable_opex*100,
        process_water_cost/variable_opex*100
    ]
})

# Sort by cost
cost_data = cost_data.sort_values('Annual Cost', ascending=True)

fig = go.Figure()

fig.add_trace(go.Bar(
    y=cost_data['Cost Category'],
    x=cost_data['Annual Cost'],
    orientation='h',
    text=[f'${x/1e6:.1f}M ({p:.0f}%)' for x, p in zip(cost_data['Annual Cost'], cost_data['Percent of Total'])],
    textposition='outside',
    marker=dict(
        color=cost_data['Annual Cost'],
        colorscale='Reds',
        showscale=False
    )
))

fig.update_layout(
    title='Annual Variable Operating Cost Breakdown',
    xaxis_title='Annual Cost (USD)',
    yaxis_title='Cost Category',
    height=400
)

max_value = cost_data['Annual Cost'].max()
fig.update_xaxes(range=[0, max_value * 1.1])
fig.show()

print(f"\n📊 Cost Analysis:")
print(f"   Electricity is {electricity_cost/variable_opex*100:.1f}% of variable operating costs")
print(f"   Total variable OpEx: ${variable_opex/1e6:.2f}M")
print(f"   Gross Margin: {gross_margin/total_revenue*100:.1f}%")
print(f"\n💡 Key Insight:")
print(f"   Electricity is the dominant operating cost. Process economics are")
print(f"   likely sensitive to electricity prices and availability of renewable power.")


📊 Cost Analysis:
   Electricity is 74.3% of variable operating costs
   Total variable OpEx: $10.87M
   Gross Margin: 59.7%

💡 Key Insight:
   Electricity is the dominant operating cost. Process economics are
   likely sensitive to electricity prices and availability of renewable power.


In [1604]:
# Create waterfall showing revenue to profit flow
categories = [
    'H₂ Revenue',
    'Acetylene Revenue',
    'RNG Cost',
    'Electricity Cost',
    'Water Costs',
    'Gross Margin'
]

values = [
    h2_revenue,
    cb_revenue,
    -rng_cost,
    -electricity_cost,
    -(cooling_water_cost + process_water_cost),
    0  # Will be calculated as total
]

fig = go.Figure(go.Waterfall(
    name="Annual Economics",
    orientation="v",
    measure=["relative", "relative", "relative", "relative", "relative", "total"],
    x=categories,
    y=values[:-1] + [gross_margin],
    text=[f'${v/1e6:+.2f}M' if v != 0 else f'${gross_margin/1e6:.2f}M' for v in values[:-1] + [gross_margin]],
    textposition="outside",
    connector={"line": {"color": "rgb(63, 63, 63)"}},
    increasing={"marker": {"color": "#2ecc71"}},
    decreasing={"marker": {"color": "#e74c3c"}},
    totals={"marker": {"color": "#3498db"}}
))

fig.update_layout(
    title="Annual Revenue to Gross Margin (Waterfall)",
    yaxis_title="Amount (USD)",
    showlegend=False,
    height=500
)

fig.show()

print(f"\n📊 Annual Economics Summary:")
print(f"   Total Revenue:        ${total_revenue/1e6:>6.2f}M")
print(f"   Variable Costs:       ${variable_opex/1e6:>6.2f}M")
print(f"   ─────────────────────────────────")
print(f"   Gross Margin:         ${gross_margin/1e6:>6.2f}M")
print(f"   Gross Margin %:       {gross_margin/total_revenue*100:>6.1f}%")
print(f"\n   Note: This excludes fixed costs (labor, maintenance, overhead)")


📊 Annual Economics Summary:
   Total Revenue:        $ 27.00M
   Variable Costs:       $ 10.87M
   ─────────────────────────────────
   Gross Margin:         $ 16.12M
   Gross Margin %:         59.7%

   Note: This excludes fixed costs (labor, maintenance, overhead)


## Comparison with Steam Methane Reforming (SMR)

Steam methane reforming is the incumbent technology for hydrogen production, accounting for ~96% of global production. Let's compare our process:

In [1605]:
# SMR baseline data (from performance_metrics.txt)
comparison_data = {
    'Metric': [
        'Water Consumption (kg/kg H₂)',
        'CO₂ Emissions (kg/kg H₂)',
        'Energy Consumption (kJ/kg H₂)',
        'H₂ Production Cost ($/kg)*'
    ],
    'Steam Methane Reforming': [
        4.5,      # Water
        9.7,      # CO2
        27,       # Energy
        2.50      # Cost (typical)
    ],
    'Microwave Plasma Process': [
        h2o_per_h2,          # XX
        co2_per_h2,          # XX
        energy_per_h2,       # Higher but renewable
        lcoh                 # XX
    ]
}

comparison_df = pd.DataFrame(comparison_data)

# Calculate improvements
comparison_df['Improvement (%)'] = (
    (comparison_df['Steam Methane Reforming'] - comparison_df['Microwave Plasma Process']) / 
    comparison_df['Steam Methane Reforming'] * 100
).round(1)

# Display with styling
display(comparison_df.style.set_properties(**{'text-align': 'center'}).set_table_styles(
    [dict(selector='th', props=[('text-align', 'center')])]
).format({
    'Steam Methane Reforming': '{:.1f}',
    'Microwave Plasma Process': '{:.2f}',
    'Improvement (%)': '{:+.1f}%'
}))

print(f"\n🌟 Environmental Advantages:")
print(f"   • Water Consumption:  {(1 - h2o_per_h2/4.5)*100:.1f}% reduction")
print(f"   • CO₂ Emissions:      {(1 - co2_per_h2/9.7)*100:.1f}% reduction")
print(f"   • Near-zero emissions when using renewable electricity")
print(f"\n⚡ Energy Trade-off:")
print(f"   • Higher energy intensity, ideally from renewable sources")
print(f"   • Our process: Higher energy input, zero carbon output")
print(f"\n💰 Cost Competitiveness:")
print(f"   • LCOH: ${lcoh:.2f}/kg vs ~$2.50/kg for SMR")
print(f"   • Higher cost, even accounting for high-value co-product")
print(f"   • Clean hydrogen incentives would likely make this competitive")

# Visualize key metrics
fig = make_subplots(
    rows=1, cols=4,
    subplot_titles=('Water Consumption', 'CO₂ Emissions', 'Energy Consumption', 'Production Cost'),
    specs=[[{'type': 'bar'}, {'type': 'bar'}, {'type': 'bar'}, {'type': 'bar'}]]
)

processes = ['SMR', 'Microwave Plasma']
colors = ['#e74c3c', '#2ecc71']

# Water
fig.add_trace(
    go.Bar(x=processes, y=[4.5, h2o_per_h2], name='Water', marker_color=colors, showlegend=False),
    row=1, col=1
)

# CO2
fig.add_trace(
    go.Bar(x=processes, y=[9.7, co2_per_h2], name='CO2', marker_color=colors, showlegend=False),
    row=1, col=2
)

# energy
fig.add_trace(
    go.Bar(x=processes, y=[27, energy_per_h2], name='Energy', marker_color=colors, showlegend=False),
    row=1, col=3
)

# Cost
fig.add_trace(
    go.Bar(x=processes, y=[2.50, lcoh], name='Cost', marker_color=colors, showlegend=False),
    row=1, col=4
)

fig.update_yaxes(title_text="kg H₂O / kg H₂", row=1, col=1)
fig.update_yaxes(title_text="kg CO₂ / kg H₂", row=1, col=2)
fig.update_yaxes(title_text="kWh / kg H₂", row=1, col=3)
fig.update_yaxes(title_text="$ / kg H₂", row=1, col=4)

fig.update_layout(height=400, showlegend=False, title_text="Process Comparison: SMR vs Microwave Plasma")

fig.show()

print(f"\n* SMR cost varies by region and natural gas prices. $2.50/kg is typical for US Gulf Coast.")


🌟 Environmental Advantages:
   • Water Consumption:  85.8% reduction
   • CO₂ Emissions:      99.9% reduction
   • Near-zero emissions when using renewable electricity

⚡ Energy Trade-off:
   • Higher energy intensity, ideally from renewable sources
   • Our process: Higher energy input, zero carbon output

💰 Cost Competitiveness:
   • LCOH: $3.13/kg vs ~$2.50/kg for SMR
   • Higher cost, even accounting for high-value co-product
   • Clean hydrogen incentives would likely make this competitive



* SMR cost varies by region and natural gas prices. $2.50/kg is typical for US Gulf Coast.


In [1606]:
def calculate_project_npv(capex, annual_revenue, annual_variable_costs, 
                          discount_rate, project_life, 
                          annual_fixed_costs=0, tax_rate=0.21):
    """
    Calculate NPV and IRR with more detailed cash flow model
    
    Parameters match actual thesis financial structure
    """
    
    # Year 0: Capital investment
    cash_flows = [-capex]
    
    # Years 1-20: Operating cash flows
    for year in range(1, project_life + 1):
        # Revenue
        revenue = annual_revenue
        
        # Costs
        variable_cost = annual_variable_costs
        fixed_cost = annual_fixed_costs
        
        # EBITDA
        ebitda = revenue - variable_cost - fixed_cost
        
        # Depreciation (simplified - straight line over 7 years)
        if year <= 7:
            depreciation = capex / 7
        else:
            depreciation = 0
        
        # EBIT
        ebit = ebitda - depreciation
        
        # Taxes
        taxes = max(0, ebit * tax_rate)
        
        # Net Income
        net_income = ebit - taxes
        
        # Cash Flow = Net Income + Depreciation (add back non-cash)
        cash_flow = net_income + depreciation
        
        cash_flows.append(cash_flow)
    
    # Calculate NPV
    npv_calc = npf.npv(discount_rate, cash_flows)
    
    # Calculate IRR
    try:
        irr_calc = npf.irr(cash_flows) * 100
    except:
        irr_calc = None
    
    # Simple payback
    cumulative = -capex
    payback_year = None
    for year in range(1, project_life + 1):
        cumulative += cash_flows[year]
        if cumulative >= 0 and payback_year is None:
            previous_cumulative = cumulative - cash_flows[year]  # Balance before this year
            fraction_of_year = abs(previous_cumulative) / cash_flows[year]
            payback_year = year - 1 + fraction_of_year
            break
    
    return {
        'NPV': npv_calc,
        'IRR': irr_calc,
        'Payback': payback_year,
        'Cash Flows': cash_flows
    }

# Calculate with actual numbers
results = calculate_project_npv(
    capex=capex,
    annual_revenue=total_revenue,
    annual_variable_costs=variable_opex,
    discount_rate=discount_rate,
    project_life=int(plant_lifetime),
    annual_fixed_costs=fixed_opex,
    tax_rate=0.21
)

print("="*80)
print("PROJECT FINANCIAL ANALYSIS")
print("="*80)
print(f"\n💰 Net Present Value:")
print(f"   NPV (15% discount):    ${results['NPV']/1e6:>8.2f} million")

print(f"\n📈 Internal Rate of Return:")
print(f"   Calculated IRR:        {results['IRR']:>8.2f}%")

print(f"\n⏱️ Payback Period:")
if results['Payback'] is None:
    print( f"   Calculated:         {'Never':>8} (doesn't pay back within {plant_lifetime} years)")
else:
    print(f"   Calculated:            {results['Payback']:>8.1f} years")

# Visualize cumulative cash flow
years = list(range(0, int(plant_lifetime) + 1))
cumulative_cf = [results['Cash Flows'][0]]  # Start with -CapEx

for i in range(1, len(results['Cash Flows'])):
    cumulative_cf.append(cumulative_cf[-1] + results['Cash Flows'][i])

fig = go.Figure()

fig.add_trace(go.Scatter(
    x=years,
    y=cumulative_cf,
    mode='lines+markers',
    name='Cumulative Cash Flow',
    fill='tozeroy',
    line=dict(width=3, color='#2ecc71'),
    marker=dict(size=6)
))

fig.add_hline(y=0, line_dash="dash", line_color="red", annotation_text="Break Even")

# Mark payback point
if results['Payback']:
    fig.add_vline(x=results['Payback'], line_dash="dot", line_color="orange",
                  annotation_text=f"Payback: {results['Payback']:.1f} years")

fig.update_layout(
    title='Cumulative Cash Flow Over Project Life',
    xaxis_title='Year',
    yaxis_title='Cumulative Cash Flow (USD)',
    hovermode='x unified',
    height=500
)

fig.show()

print(f"\n🎯 Investment Attractiveness:")
if results['NPV'] > 0:
    print(f"   ✅ Positive NPV: Project creates value")
if results['IRR'] and results['IRR'] > discount_rate * 100:
    print(f"   ✅ IRR ({results['IRR']:.1f}%) exceeds discount rate ({discount_rate*100}%)")
if results['Payback'] and results['Payback'] < 5:
    print(f"   ✅ Fast payback: {results['Payback']:.1f} years is excellent for chemical industry")

PROJECT FINANCIAL ANALYSIS

💰 Net Present Value:
   NPV (15% discount):    $    9.65 million

📈 Internal Rate of Return:
   Calculated IRR:           18.99%

⏱️ Payback Period:
   Calculated:                 4.9 years



🎯 Investment Attractiveness:
   ✅ Positive NPV: Project creates value
   ✅ IRR (19.0%) exceeds discount rate (15.0%)
   ✅ Fast payback: 4.9 years is excellent for chemical industry


In [1607]:
# Sensitivity analysis on key parameters
def sensitivity_analysis_tornado(base_npv, base_params):
    """
    Run sensitivity analysis varying one parameter at a time
    Calculate impact on NPV
    """
    
    results = []
    
    # Parameters to test from sensitivity_ranges
    params_to_test = sensitivity_ranges['parameter'].tolist()
    
    for param in params_to_test:
        row = sensitivity_ranges[sensitivity_ranges['parameter'] == param].iloc[0]
        
        low_val = row['low_value']
        high_val = row['high_value']
        base_val = row['base_value']
        
        # Calculate NPV at low and high values
        # This is simplified - in reality you'd recalculate full model
        # For demonstration, we'll estimate impact
        
        if param == 'electricity_price':
            # Electricity affects variable costs
            elec_cost_base = hydrogen_production * energy_per_h2 * base_val
            elec_cost_low = hydrogen_production * energy_per_h2 * low_val
            elec_cost_high = hydrogen_production * energy_per_h2 * high_val
            
            delta_low = elec_cost_base - elec_cost_low
            delta_high = elec_cost_high - elec_cost_base
            
            # NPV impact (simplified: annual savings * annuity factor)
            annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
            npv_low = base_npv + delta_low * annuity_factor
            npv_high = base_npv - delta_high * annuity_factor
            
        elif param == 'hydrogen_price':
            # H2 price affects revenue
            revenue_change_low = (low_val - base_val) * hydrogen_production
            revenue_change_high = (high_val - base_val) * hydrogen_production
            
            annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
            npv_low = base_npv + revenue_change_low * annuity_factor
            npv_high = base_npv + revenue_change_high * annuity_factor
            
        elif param == 'acetylene_black_price':
            # CB price affects revenue
            revenue_change_low = (low_val - base_val) * acetylene_black_production
            revenue_change_high = (high_val - base_val) * acetylene_black_production
            
            annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
            npv_low = base_npv + revenue_change_low * annuity_factor
            npv_high = base_npv + revenue_change_high * annuity_factor
            
        elif param == 'capex_total':
            # CapEx directly affects NPV
            npv_low = base_npv + (base_val - low_val)
            npv_high = base_npv + (base_val - high_val)
            
        elif param == 'rng_price':
            # RNG price affects costs
            rng_annual_consumption = 1923 * annual_operating_days * 24
            cost_change_low = (low_val - base_val) * rng_annual_consumption
            cost_change_high = (high_val - base_val) * rng_annual_consumption
            
            annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
            npv_low = base_npv - cost_change_low * annuity_factor
            npv_high = base_npv - cost_change_high * annuity_factor
            
        else:
            # For other parameters, use rough estimate
            npv_low = base_npv * 0.9
            npv_high = base_npv * 1.1
        
        results.append({
            'Parameter': param.replace('_', ' ').title(),
            'NPV_Low': npv_low,
            'NPV_High': npv_high,
            'Range': abs(npv_high - npv_low),
            'Impact_Low': npv_low - base_npv,
            'Impact_High': npv_high - base_npv
        })
    
    return pd.DataFrame(results).sort_values('Range', ascending=True)

# Run sensitivity analysis
sensitivity_results = sensitivity_analysis_tornado(results['NPV'], {
    'electricity_price': electricity_price,
    'hydrogen_price': h2_price,
    'acetylene_black_price': cb_price,
    'capex_total': capex,
    'rng_price': rng_price
})

# Create tornado diagram
fig = go.Figure()

# Add bars for each parameter
for idx, row in sensitivity_results.iterrows():
    fig.add_trace(go.Bar(
        name=row['Parameter'],
        y=[row['Parameter']],
        x=[row['Impact_Low']],
        orientation='h',
        marker=dict(color='#3498db'),
        showlegend=False,
        hovertemplate=f"{row['Parameter']}<br>Low Case: ${row['NPV_Low']/1e6:.1f}M<extra></extra>"
    ))
    
    fig.add_trace(go.Bar(
        name=row['Parameter'],
        y=[row['Parameter']],
        x=[row['Impact_High']],
        orientation='h',
        marker=dict(color='#e74c3c'),
        showlegend=False,
        hovertemplate=f"{row['Parameter']}<br>High Case: ${row['NPV_High']/1e6:.1f}M<extra></extra>"
    ))

fig.add_vline(x=0, line_width=2, line_dash="solid", line_color="black")

fig.update_layout(
    title='Sensitivity Analysis: Impact on NPV (Tornado Diagram)',
    xaxis_title='Change in NPV from Base Case ($)',
    yaxis_title='Parameter',
    barmode='overlay',
    height=500,
    hovermode='y unified'
)

fig.show()

print("\n📊 Sensitivity Analysis Results:")
print("\nParameters ranked by impact on NPV (most to least sensitive):")
print("\n" + "="*80)
for idx, row in sensitivity_results.sort_values('Range', ascending=False).iterrows():
    print(f"{row['Parameter']:.<40} Range: ${row['Range']/1e6:>6.1f}M")
print("="*80)

print(f"\n💡 Key Insights:")
most_sensitive = sensitivity_results.sort_values('Range', ascending=False).iloc[0]
print(f"   • Most sensitive parameter: {most_sensitive['Parameter']}")
print(f"   • NPV range: ${most_sensitive['NPV_Low']/1e6:.1f}M to ${most_sensitive['NPV_High']/1e6:.1f}M")
print(f"   • Project remains attractive across most scenarios if NPV > 0")


📊 Sensitivity Analysis Results:

Parameters ranked by impact on NPV (most to least sensitive):

Acetylene Black Price................... Range: $  82.3M
Hydrogen Price.......................... Range: $  78.5M
Electricity Price....................... Range: $  43.4M
Capex Total............................. Range: $  30.0M
Rng Price............................... Range: $  11.5M
Carbon Conversion....................... Range: $   1.9M
Capacity Factor......................... Range: $   1.9M
Discount Rate........................... Range: $   1.9M

💡 Key Insights:
   • Most sensitive parameter: Acetylene Black Price
   • NPV range: $-31.5M to $50.8M
   • Project remains attractive across most scenarios if NPV > 0


In [1608]:
# Detailed scenario analysis for electricity price
# This is the most sensitive variable and represents renewable energy availability

electricity_scenarios = {
    'Scenario': [
        'Very Low\n(Abundant Renewables)',
        'Low\n(TX/Midwest Wind)',
        'Base Case\n(Industrial Average)',
        'High\n(CA/Northeast)',
        'Very High\n(Peak/Premium)'
    ],
    'Electricity Price ($/kWh)': [0.03, 0.05, 0.07, 0.10, 0.12],
    'Context': [
        'West Texas wind, overnight rates',
        'PPA with wind farm',
        'Standard industrial rate',
        'High-cost regions',
        'Peak demand pricing'
    ]
}

scenario_df = pd.DataFrame(electricity_scenarios)

# Calculate metrics for each scenario
lcoh_scenarios = []
npv_scenarios = []
margin_scenarios = []

for elec_price in scenario_df['Electricity Price ($/kWh)']:
    # Recalculate electricity cost
    annual_elec = hydrogen_production * energy_per_h2 * elec_price
    
    # Recalculate variable costs
    new_variable_costs = (rng_cost + annual_elec + cooling_water_cost + process_water_cost)
    
    # Recalculate margin
    new_margin = total_revenue - new_variable_costs
    margin_pct = new_margin / total_revenue * 100
    
    # Estimate NPV impact
    cost_delta = annual_elec - electricity_cost
    annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
    new_npv = results['NPV'] - cost_delta * annuity_factor
    
    # LCOH estimate
    # Simplified: base LCOH + electricity cost change per kg H2
    lcoh_delta = (elec_price - electricity_price) * energy_per_h2
    new_lcoh = lcoh + lcoh_delta
    
    lcoh_scenarios.append(new_lcoh)
    npv_scenarios.append(new_npv)
    margin_scenarios.append(margin_pct)

scenario_df['LCOH ($/kg)'] = lcoh_scenarios
scenario_df['NPV ($M)'] = [x/1e6 for x in npv_scenarios]
scenario_df['Gross Margin (%)'] = margin_scenarios

# Display table
display(scenario_df[['Scenario', 'Electricity Price ($/kWh)', 'LCOH ($/kg)', 
                     'NPV ($M)', 'Gross Margin (%)']].style.format({
    'Electricity Price ($/kWh)': '{:.2f}',
    'LCOH ($/kg)': '{:.2f}',
    'NPV ($M)': '{:.1f}',
    'Gross Margin (%)': '{:.1f}%'
}).set_properties(**{'text-align': 'center'}))

# Visualize
fig = make_subplots(
    rows=1, cols=2,
    subplot_titles=('LCOH vs Electricity Price', 'NPV vs Electricity Price')
)

fig.add_trace(
    go.Scatter(
        x=scenario_df['Electricity Price ($/kWh)'],
        y=scenario_df['LCOH ($/kg)'],
        mode='lines+markers',
        name='LCOH',
        line=dict(width=3, color='#3498db'),
        marker=dict(size=10)
    ),
    row=1, col=1
)

# Add H2 market price reference
fig.add_hline(y=h2_price, line_dash="dash", line_color="green", 
              annotation_text=f"H₂ Market Price (${h2_price}/kg)",
              row=1, col=1)

# add SMR LCOH
fig.add_hline(y=2.50, line_dash="dash", line_color="red",
              annotation_text="SMR LCOH ($2.50/kg)",
              row=1, col=1)

fig.add_trace(
    go.Scatter(
        x=scenario_df['Electricity Price ($/kWh)'],
        y=scenario_df['NPV ($M)'],
        mode='lines+markers',
        name='NPV',
        line=dict(width=3, color='#e74c3c'),
        marker=dict(size=10)
    ),
    row=1, col=2
)

fig.add_hline(y=0, line_dash="dash", line_color="red",
              annotation_text="Break Even",
              row=1, col=2)

fig.update_xaxes(title_text="Electricity Price ($/kWh)", row=1, col=1)
fig.update_xaxes(title_text="Electricity Price ($/kWh)", row=1, col=2)
fig.update_yaxes(title_text="LCOH ($/kg H₂)", row=1, col=1)
fig.update_yaxes(title_text="NPV ($ millions)", row=1, col=2)

fig.update_layout(height=400, showlegend=False, title_text="Electricity Price Scenario Analysis")

fig.show()

print(f"\n💡 Strategic Insights:")
print(f"\n🌟 Best Case (Abundant Renewables at $0.03/kWh):")
print(f"   • LCOH: ${lcoh_scenarios[0]:.2f}/kg")
print(f"   • NPV: ${npv_scenarios[0]/1e6:.1f}M")
print(f"   • Project is HIGHLY attractive")

print(f"\n⚠️ Worst Case (Premium Electricity at $0.12/kWh):")
print(f"   • LCOH: ${lcoh_scenarios[-1]:.2f}/kg")
print(f"   • NPV: ${npv_scenarios[-1]/1e6:.1f}M")
worst_viable = "still viable" if npv_scenarios[-1] > 0 else "NOT viable"
print(f"   • Project is {worst_viable}")

print(f"\n🎯 Optimal Deployment Strategy:")
print(f"   • Target regions with low-cost renewable electricity")
print(f"   • Co-locate with wind/solar farms for direct PPAs")
print(f"   • Consider regions: West Texas, Midwest, Pacific Northwest")
print(f"   • Avoid: High electricity cost regions unless H₂ prices are premium")


💡 Strategic Insights:

🌟 Best Case (Abundant Renewables at $0.03/kWh):
   • LCOH: $1.66/kg
   • NPV: $38.6M
   • Project is HIGHLY attractive

⚠️ Worst Case (Premium Electricity at $0.12/kWh):
   • LCOH: $4.97/kg
   • NPV: $-26.5M
   • Project is NOT viable

🎯 Optimal Deployment Strategy:
   • Target regions with low-cost renewable electricity
   • Co-locate with wind/solar farms for direct PPAs
   • Consider regions: West Texas, Midwest, Pacific Northwest
   • Avoid: High electricity cost regions unless H₂ prices are premium


In [1609]:
# Monte Carlo simulation for uncertainty analysis
np.random.seed(42)
n_simulations = 5000

print("Running Monte Carlo simulation with 5,000 iterations...")
print("This quantifies project risk by sampling from parameter distributions\n")

# Storage for results
npv_results = []
irr_results = []
lcoh_results = []
margin_results = []

for i in range(n_simulations):
    # Sample parameters from distributions
    # Using normal distribution with std = (high - low) / 6 (capturing 99.7% within range)
    
    elec_price_sim = np.random.normal(
        electricity_price, 
        (sensitivity_ranges[sensitivity_ranges['parameter'] == 'electricity_price']['high_value'].values[0] - 
         sensitivity_ranges[sensitivity_ranges['parameter'] == 'electricity_price']['low_value'].values[0]) / 6)
    
    elec_price_sim = max(0.03, min(0.15, elec_price_sim))  # Bound to reasonable range

    h2_price_sim = np.random.normal(
        h2_price,
        (5.0 - 2.0) / 6
    )
    h2_price_sim = max(1.5, min(6.0, h2_price_sim))

    cb_price_sim = np.random.normal(
        cb_price,
        (3.0 - 1.0) / 6
    )
    cb_price_sim = max(0.5, min(3.5, cb_price_sim))

    rng_price_sim = np.random.normal(
        rng_price,
        (0.25 - 0.10) / 6
    )
    rng_price_sim = max(0, min(0.30, rng_price_sim))

    capex_sim = np.random.normal(
        capex,
        (capex * 0.2) / 3  # ±20% range
    )
    capex_sim = max(capex * 0.7, min(capex * 1.3, capex_sim))

    capacity_factor_sim = np.random.normal(
        capacity_factor,
        (0.95 - 0.85) / 6
    )
    capacity_factor_sim = max(0.80, min(0.98, capacity_factor_sim))

    # Recalculate metrics with sampled parameters
    h2_annual_sim = h2_rate * 365 * 24 * capacity_factor_sim
    cb_annual_sim = cb_rate * 365 * 24 * capacity_factor_sim

    # Revenue
    h2_revenue_sim = h2_annual_sim * h2_price_sim
    cb_revenue_sim = cb_annual_sim * cb_price_sim
    total_revenue_sim = h2_revenue_sim + cb_revenue_sim

    # Costs
    rng_annual_consumption = 1923 * annual_operating_days * 24
    rng_cost_sim = rng_annual_consumption * rng_price_sim
    elec_cost_sim = h2_annual_sim * energy_per_h2 * elec_price_sim
    variable_costs_sim = rng_cost_sim + elec_cost_sim + cooling_water_cost + process_water_cost

    # Margin
    margin_sim = total_revenue_sim - variable_costs_sim
    margin_pct_sim = margin_sim / total_revenue_sim * 100

    # LCOH (simplified)
    total_annual_cost_sim = (capex_sim * ((discount_rate * (1+discount_rate)**plant_lifetime) / 
                                          ((1+discount_rate)**plant_lifetime - 1))) + variable_costs_sim
    lcoh_sim = (total_annual_cost_sim - cb_revenue_sim) / h2_annual_sim

    # NPV (simplified)
    annual_cf = margin_sim
    annuity_factor = (1 - (1 + discount_rate)**(-plant_lifetime)) / discount_rate
    npv_sim = -capex_sim + annual_cf * annuity_factor

    # IRR (approximation)
    if annual_cf > 0:
        irr_sim = (annual_cf / capex_sim) * 100
    else:
        irr_sim = -100

    npv_results.append(npv_sim)
    irr_results.append(irr_sim)
    lcoh_results.append(lcoh_sim)
    margin_results.append(margin_pct_sim)

# convert to arrays

npv_results = np.array(npv_results)
irr_results = np.array(irr_results)
lcoh_results = np.array(lcoh_results)
margin_results = np.array(margin_results)

print("="*80)
print("MONTE CARLO UNCERTAINTY ANALYSIS RESULTS (5,000 iterations)")
print("="*80)
print(f"\n💰 NET PRESENT VALUE:")
print(f"   Mean:              ${np.mean(npv_results)/1e6:>8.2f} million")
print(f"   Median:            ${np.median(npv_results)/1e6:>8.2f} million")
print(f"   Std Deviation:     ${np.std(npv_results)/1e6:>8.2f} million")
print(f"   5th Percentile:    ${np.percentile(npv_results, 5)/1e6:>8.2f} million")
print(f"   95th Percentile:   ${np.percentile(npv_results, 95)/1e6:>8.2f} million")
print(f"   Probability NPV>0: {(npv_results > 0).sum() / len(npv_results) * 100:>7.1f}%")
print(f"\n💵 LEVELIZED COST OF HYDROGEN:")
print(f"   Mean:              ${np.mean(lcoh_results):>8.2f}/kg")
print(f"   Median:            ${np.median(lcoh_results):>8.2f}/kg")
print(f"   Std Deviation:     ${np.std(lcoh_results):>8.2f}/kg")
print(f"   5th Percentile:    ${np.percentile(lcoh_results, 5):>8.2f}/kg")
print(f"   95th Percentile:   ${np.percentile(lcoh_results, 95):>8.2f}/kg")
print(f"   Probability LCOH<$3.18: {(lcoh_results < h2_price).sum() / len(lcoh_results) * 100:>4.1f}%")
print(f"\n📊 GROSS MARGIN:")
print(f"   Mean:              {np.mean(margin_results):>8.1f}%")
print(f"   Median:            {np.median(margin_results):>8.1f}%")
print(f"   5th Percentile:    {np.percentile(margin_results, 5):>8.1f}%")
print(f"   95th Percentile:   {np.percentile(margin_results, 95):>8.1f}%")
print(f"   Probability Margin>20%: {(margin_results > 20).sum() / len(margin_results) * 100:>4.1f}%")

#visualize distributions
fig = make_subplots(
rows=2, cols=2,
subplot_titles=('NPV Distribution', 'LCOH Distribution',
'Gross Margin Distribution', 'NPV Probability'),
specs=[[{'type': 'histogram'}, {'type': 'histogram'}],
[{'type': 'histogram'}, {'type': 'scatter'}]]
)

#npv histogram
fig.add_trace(
go.Histogram(x=npv_results/1e6, name='NPV', marker_color='#3498db',
nbinsx=50, showlegend=False),
row=1, col=1
)
fig.add_vline(x=npv/1e6, line_dash="dash", line_color="red",
annotation_text="Base Case", row=1, col=1)

#lcoh histogram
fig.add_trace(
go.Histogram(x=lcoh_results, name='LCOH', marker_color='#2ecc71',
nbinsx=50, showlegend=False),
row=1, col=2
)
fig.add_vline(x=lcoh, line_dash="dash", line_color="red",
annotation_text="Base Case", row=1, col=2)
fig.add_vline(x=h2_price, line_dash="dot", line_color="green",
annotation_text="Market Price", row=1, col=2)

#margin histogram
fig.add_trace(
go.Histogram(x=margin_results, name='Margin', marker_color='#e74c3c',
nbinsx=50, showlegend=False),
row=2, col=1
)
fig.add_vline(x=gross_margin/total_revenue*100, line_dash="dash", line_color="red",
annotation_text="Base Case", row=2, col=1)

#npv cumulative probability
sorted_npv = np.sort(npv_results)
cumulative_prob = np.arange(1, len(sorted_npv) + 1) / len(sorted_npv) * 100
fig.add_trace(
go.Scatter(x=sorted_npv/1e6, y=cumulative_prob,
mode='lines', line=dict(width=3, color='#9b59b6'),
showlegend=False),
row=2, col=2
)
fig.add_vline(x=0, line_dash="dash", line_color="red",
annotation_text="Break Even", row=2, col=2)
fig.update_xaxes(title_text="NPV ($ millions)", row=1, col=1)
fig.update_xaxes(title_text="LCOH ($/kg H₂)", row=1, col=2)
fig.update_xaxes(title_text="Gross Margin (%)", row=2, col=1)
fig.update_xaxes(title_text="NPV ($ millions)", row=2, col=2)
fig.update_yaxes(title_text="Frequency", row=1, col=1)
fig.update_yaxes(title_text="Frequency", row=1, col=2)
fig.update_yaxes(title_text="Frequency", row=2, col=1)
fig.update_yaxes(title_text="Cumulative Probability (%)", row=2, col=2)
fig.update_layout(height=700, title_text="Monte Carlo Uncertainty Analysis")
fig.show()

#percentile analysis
percentiles = [5, 10, 25, 50, 75, 90, 95]
print("\n📈 Percentile Analysis:")
print("\nNPV ($ millions):")
for p in percentiles:
    print(f"   {p:>2}th percentile: ${np.percentile(npv_results, p)/1e6:>6.1f}M")
    print("\nLCOH ($/kg H₂):")
for p in percentiles:
    print(f"   {p:>2}th percentile: ${np.percentile(lcoh_results, p):>5.2f}")
    print(f"\n🎯 Risk Assessment:")
    print(f"   • {(npv_results > 0).sum() / len(npv_results) * 100:.1f}% probability of positive NPV")
    print(f"   • {(lcoh_results < h2_price).sum() / len(lcoh_results) * 100:.1f}% probability LCOH below market price")
    print(f"   • {(margin_results > 20).sum() / len(margin_results) * 100:.1f}% probability of healthy margin (>20%)")
if (npv_results > 0).sum() / len(npv_results) > 0.7:
    print(f"\n   ✅ Project shows STRONG robustness to uncertainty")
elif (npv_results > 0).sum() / len(npv_results) > 0.5:
    print(f"\n   ⚠️ Project shows MODERATE robustness to uncertainty")
else:
    print(f"\n   ❌ Project shows HIGH risk under uncertainty")

Running Monte Carlo simulation with 5,000 iterations...
This quantifies project risk by sampling from parameter distributions

MONTE CARLO UNCERTAINTY ANALYSIS RESULTS (5,000 iterations)

💰 NET PRESENT VALUE:
   Mean:              $   64.96 million
   Median:            $   64.99 million
   Std Deviation:     $   20.08 million
   5th Percentile:    $   32.22 million
   95th Percentile:   $   97.50 million
   Probability NPV>0:    99.9%

💵 LEVELIZED COST OF HYDROGEN:
   Mean:              $    0.92/kg
   Median:            $    0.94/kg
   Std Deviation:     $    0.82/kg
   5th Percentile:    $   -0.42/kg
   95th Percentile:   $    2.26/kg
   Probability LCOH<$3.18: 100.0%

📊 GROSS MARGIN:
   Mean:                  60.0%
   Median:                60.4%
   5th Percentile:        49.4%
   95th Percentile:       69.3%
   Probability Margin>20%: 100.0%



📈 Percentile Analysis:

NPV ($ millions):
    5th percentile: $  32.2M

LCOH ($/kg H₂):
   10th percentile: $  38.8M

LCOH ($/kg H₂):
   25th percentile: $  51.5M

LCOH ($/kg H₂):
   50th percentile: $  65.0M

LCOH ($/kg H₂):
   75th percentile: $  78.8M

LCOH ($/kg H₂):
   90th percentile: $  90.4M

LCOH ($/kg H₂):
   95th percentile: $  97.5M

LCOH ($/kg H₂):
    5th percentile: $-0.42

🎯 Risk Assessment:
   • 99.9% probability of positive NPV
   • 100.0% probability LCOH below market price
   • 100.0% probability of healthy margin (>20%)
   10th percentile: $-0.15

🎯 Risk Assessment:
   • 99.9% probability of positive NPV
   • 100.0% probability LCOH below market price
   • 100.0% probability of healthy margin (>20%)
   25th percentile: $ 0.37

🎯 Risk Assessment:
   • 99.9% probability of positive NPV
   • 100.0% probability LCOH below market price
   • 100.0% probability of healthy margin (>20%)
   50th percentile: $ 0.94

🎯 Risk Assessment:
   • 99.9% probability of positive NPV


## Geographic Deployment Strategy

The economics of this process vary significantly by region based on electricity costs and hydrogen demand. Let's analyze optimal deployment locations.

In [1610]:
# Regional analysis based on electricity prices and hydrogen markets
regions = {
    'Region': [
        'West Texas',
        'Midwest (IA/MN)',
        'Pacific Northwest',
        'California',
        'Northeast',
        'Southeast'
    ],
    'Electricity Price ($/kWh)': [0.045, 0.055, 0.060, 0.180, 0.140, 0.085],
    'H₂ Demand Level': ['Medium', 'Low', 'Medium', 'High', 'High', 'Medium'],
    'Renewable Availability': ['Excellent (Wind)', 'Excellent (Wind)', 'Excellent (Hydro)',
                                'Good (Solar)', 'Moderate', 'Moderate'],
    'Key Industries': ['Refining, Chemicals', 'Agriculture, Food', 'Aerospace, Tech',
                       'Refining, Transport', 'Manufacturing', 'Chemicals, Aerospace']
}

regions_df = pd.DataFrame(regions)

# Calculate LCOH for each region
regions_df['LCOH ($/kg)'] = regions_df['Electricity Price ($/kWh)'].apply(
    lambda e: lcoh + (e - electricity_price) * energy_per_kg_h2
)

# Calculate NPV impact
regions_df['NPV ($M)'] = regions_df['Electricity Price ($/kWh)'].apply(
    lambda e: (npv - (e - electricity_price) * energy_per_kg_h2 * h2_annual * 
              ((1 - (1+discount_rate)**(-plant_lifetime)) / discount_rate)) / 1e6
)

# Create a transformed NPV for sizing (shift all values to be positive)
regions_df['NPV_for_size'] = regions_df['NPV ($M)'] - regions_df['NPV ($M)'].min() + 1

# Ranking
regions_df['Economic Rank'] = regions_df['NPV ($M)'].rank(ascending=False).astype(int)

# Display
display(regions_df.style.format({
    'Electricity Price ($/kWh)': '{:.3f}',
    'LCOH ($/kg)': '{:.2f}',
    'NPV ($M)': '{:.1f}'
}).background_gradient(subset=['NPV ($M)'], cmap='RdYlGn'))

# Visualize
fig = px.scatter(
    regions_df,
    x='Electricity Price ($/kWh)',
    y='LCOH ($/kg)',
    size='NPV_for_size',  # Use the positive-shifted values
    color='Renewable Availability',
    text='Region',
    title='Regional Economics: Electricity Price vs LCOH',
    labels={'LCOH ($/kg)': 'Levelized Cost of Hydrogen ($/kg)'},
    size_max=40,
    hover_data={'NPV ($M)': ':.1f', 'NPV_for_size': False}  # Show actual NPV in hover
)

fig.add_hline(y=h2_price, line_dash="dash", line_color="green",
              annotation_text=f"H₂ Market Price (${h2_price}/kg)")

fig.update_traces(textposition='top center')
fig.update_layout(height=500)

fig.show()

print(f"\n🌎 Optimal Deployment Strategy:\n")

top_region = regions_df.loc[regions_df['NPV ($M)'].idxmax()]
print(f"1️⃣ BEST: {top_region['Region']}")
print(f"   • Electricity: ${top_region['Electricity Price ($/kWh)']:.3f}/kWh")
print(f"   • LCOH: ${top_region['LCOH ($/kg)']:.2f}/kg")
print(f"   • NPV: ${top_region['NPV ($M)']:.1f}M")
print(f"   • Renewable Source: {top_region['Renewable Availability']}")
print(f"   • Strategy: Co-locate with wind farms, secure long-term PPA\n")

worst_region = regions_df.loc[regions_df['NPV ($M)'].idxmin()]
print(f"❌ AVOID: {worst_region['Region']}")
print(f"   • Electricity: ${worst_region['Electricity Price ($/kWh)']:.3f}/kWh")
print(f"   • LCOH: ${worst_region['LCOH ($/kg)']:.2f}/kg")
print(f"   • NPV: ${worst_region['NPV ($M)']:.1f}M")
print(f"   • High electricity costs make project marginal\n")

print(f"💡 Key Insights:")
print(f"   • Midwest and West Texas offer best economics (abundant wind)")
print(f"   • California has high demand but high electricity costs")
print(f"   • Consider pairing with on-site solar in high-cost regions")
print(f"   • RNG availability also matters - prioritize near dairy farms/landfills")


🌎 Optimal Deployment Strategy:

1️⃣ BEST: West Texas
   • Electricity: $0.045/kWh
   • LCOH: $2.21/kg
   • NPV: $23.1M
   • Renewable Source: Excellent (Wind)
   • Strategy: Co-locate with wind farms, secure long-term PPA

❌ AVOID: California
   • Electricity: $0.180/kWh
   • LCOH: $7.18/kg
   • NPV: $-73.3M
   • High electricity costs make project marginal

💡 Key Insights:
   • Midwest and West Texas offer best economics (abundant wind)
   • California has high demand but high electricity costs
   • Consider pairing with on-site solar in high-cost regions
   • RNG availability also matters - prioritize near dairy farms/landfills


## Key Findings

### Strong Financial Performance

**Base Case Economics (Conservative Pricing):**
- NPV (20 years, 15% discount): **$20.6 million**
- IRR: **85.95%** (far exceeds typical hurdle rates)
- Payback Period: **3.2 years** (excellent for chemical industry)
- Gross Margin: **30.6%** (healthy for process industry)
- LCOH: **$2.41/kg** (competitive with market price of $3.18/kg)

**Investment Attractiveness:**
- Positive NPV in >70% of Monte Carlo scenarios
- IRR significantly exceeds discount rate (15%)
- Fast payback reduces risk
- Robust to reasonable parameter uncertainty

### 2. Critical Success Factors

**Most Influential Parameters (from sensitivity analysis):**

1. **Electricity Price** - Dominant driver of economics
   - Represents 89% of variable operating costs
   - Optimal deployment in low-cost renewable regions
   - Target: <$0.07/kWh for strong economics
   
2. **Hydrogen Selling Price** - Market risk
   - Conservative base case at $3.18/kg
   - Market prices range $2-5/kg depending on grade/region
   - Premium markets (fuel cell grade, California LCFS) can exceed $5/kg

3. **Acetylene Black Price** - Co-product value
   - Provides 40% of total revenue
   - Premium conductive grade commands $2-3/kg
   - Battery market growth supports strong pricing

4. **Capital Costs** - Technology maturity
   - Current estimate: $45.6M for 3,100 tonne/year plant
   - Pentacarbon technology relatively new (scale-up risk)
   - Learning curve improvements possible with deployment

### 3. Environmental Superiority

**Dramatic improvements vs Steam Methane Reforming:**

- **CO₂ Emissions:** 99.9% reduction (0.12 vs 9.7 kg CO₂/kg H₂)
- **Water Consumption:** 86.7% reduction (0.6 vs 4.5 kg H₂O/kg H₂)
- **Renewable Energy:** 100% when using renewable electricity
- **Waste Utilization:** Converts waste RNG to valuable products

**Climate Impact:**
- Annual CO₂ avoidance: ~30,000 tonnes vs SMR
- Equivalent to removing ~6,500 cars from roads
- Supports corporate net-zero commitments

### 4. Market Positioning

**Competitive Advantages:**
- ✅ Near-zero carbon intensity (critical for CA LCFS, EU hydrogen strategy)
- ✅ Premium co-product (acetylene black for batteries)
- ✅ Waste feedstock utilization (RNG from landfills/dairy)
- ✅ Modular scale (right-sized for distributed production)
- ✅ No water stress concerns

**Competitive Challenges:**
- ⚠️ Higher energy intensity than SMR (but from renewables)
- ⚠️ Technology relatively new (limited deployment track record)
- ⚠️ Electricity price sensitivity (requires low-cost renewable access)
- ⚠️ Smaller scale than large SMR plants (3,100 vs 10,000+ tonnes/year)

### 5. Strategic Recommendations

**Near-Term (1-2 years):**

1. **Pilot Demonstration**
   - Validate ASPEN simulation with pilot-scale unit
   - Prove acetylene black quality for battery market
   - Demonstrate 8,000+ hour reliability
   - Engage potential offtakers early

2. **Target Markets**
   - Industrial customers with clean hydrogen mandates
   - California (LCFS credits, strict emissions standards)
   - Companies with net-zero commitments
   - Battery manufacturers (acetylene black market)

3. **Optimize Deployment Locations**
   - **Priority:** West Texas, Midwest (low-cost wind)
   - Co-locate with wind/solar farms for direct PPAs
   - Near RNG sources (landfills, dairy digesters)
   - Proximity to industrial hydrogen demand

4. **Secure Offtake Agreements**
   - Long-term hydrogen purchase agreements
   - Acetylene black supply contracts with battery makers
   - Lock in pricing to reduce market risk

**Medium-Term (3-5 years):**

1. **Scale Technology**
   - Build first commercial-scale plant (3,100 tonne/year)
   - Demonstrate consistent acetylene black quality
   - Achieve target capacity factor (>90%)
   - Reduce CapEx through learning curve

2. **Business Model Innovation**
   - "Hydrogen-as-a-Service" for lower customer CapEx
   - Bundled hydrogen + carbon credit sales
   - Partnership with renewable energy developers
   - Fleet deployment model (multiple standardized plants)

3. **Policy Engagement**
   - Qualify for IRA 45V hydrogen PTC ($3/kg for clean H₂)
   - California LCFS credit generation
   - RNG fuel credits
   - State-level clean hydrogen incentives

4. **Market Development**
   - Educate market on acetylene black value
   - Build relationships with battery manufacturers
   - Establish brand for premium clean hydrogen
   - Demonstrate reliable supply

**Long-Term (5+ years):**

1. **Portfolio Expansion**
   - Deploy 10-20 plants in optimal locations
   - Diversify feedstocks (biogas, renewable methane)
   - Explore adjacent markets (ammonia, methanol from H₂)
   - International markets with strong hydrogen policy

2. **Technology Enhancement**
   - Improve energy efficiency (reduce kWh/kg H₂)
   - Increase scale per unit (economies of scale)
   - Develop carbon black grade flexibility
   - Integration with renewable energy systems

3. **Financial Optimization**
   - Green bond financing for lower cost of capital
   - Carbon credit monetization strategies
   - Strategic partnerships with industrial gas companies
   - Consider yieldco structure for cash flow assets

---

## Conclusions

The Pentacarbon microwave plasma process for hydrogen and acetylene black production represents a **technically viable and economically attractive** alternative to conventional hydrogen production.

**Key Strengths:**
✅ **Strong economics** - Positive NPV, high IRR, fast payback  
✅ **Environmental leadership** - 99.9% CO₂ reduction vs SMR  
✅ **Revenue diversification** - Premium acetylene black co-product  
✅ **Waste valorization** - Utilizes renewable natural gas  
✅ **Strategic timing** - Aligns with hydrogen economy growth  

**Critical Success Factors:**
🎯 **Low-cost renewable electricity** - Must target <$0.07/kWh  
🎯 **Acetylene black market** - Must secure premium pricing  
🎯 **Policy support** - Leverage hydrogen incentives and LCFS credits  
🎯 **Technology validation** - Pilot demonstration crucial  

**The business case strengthens as:**
- Hydrogen demand grows (transportation, industry, power)
- Carbon pricing increases (making SMR more expensive)
- Battery market expands (acetylene black demand)
- Renewable electricity costs decline (improving economics)

**With continued technology development and supportive policy, this process could capture meaningful share of the growing clean hydrogen market, particularly for applications requiring truly low-carbon H₂ with strong environmental credentials.**

---

## References

1. Ritter, A., Levine, S., Cannon, S., (2022). "Natural Gas Conversion into Clean Hydrogen and Carbon for Li-ion Battery Electrodes Using Renewable Energy" University of Pennsylvania Senior Design Project.

2. U.S. Environmental Protection Agency. (2023). "Emission Factors for Greenhouse Gas Inventories."

3. International Energy Agency. (2023). "Global Hydrogen Review 2023."

4. National Renewable Energy Laboratory. (2023). "Hydrogen Production Cost Analysis."

5. U.S. Department of Energy. (2023). "Clean Hydrogen Production Tax Credit (45V) Guidance."

6. Pentacarbon, Inc. (2022). Technical specifications and process data.

---

## Contact

**Adam Ritter**  

📧 adam.h.ritter@gmail.com  
💼 [LinkedIn](https://linkedin.com/in/adam-ritter-env)  
🐙 [GitHub](https://github.com/Adam-Ritter)  
🌐 [Portfolio](your-portfolio-link)

---

*This interactive analysis is based on our 2022 senior design project at Penn. The ASPEN Plus simulation and core technical data remain unchanged. Financial assumptions have been updated where appropriate to reflect 2024 market conditions.*